packages/docs/src/styles/_sections.scss

```diff
@@ -244,7 +244,9 @@
 }
 
 #{section-id("components.table-js")} {
-  .kss-example {
```

Comment:
instead of modifying this class, we can just modify the bp-table-container class directly with the same effect